mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraAttachmentDAO.java

```diff
@@ -48,16 +52,21 @@
 
 public class CassandraAttachmentDAO {
 
+    public static final int SELECT_ALL_TIMEOUT = toIntExact(TimeUnit.DAYS.toMillis(1));
```

Comment:
what about making it configurable? If it's not sufficient, I would like not to have to wait also for a merge request to change it